### ISSUE TITLE
User PATH env variable gets overwritten on windows installation if PATH is more than 1024 characters

### ISSUE THREAD

**Issue by prateekbaheti** [getgauge/gauge#150]
This occurs as mentioned in the [nullsoft installer page](http://nsis.sourceforge.net/Path_Manipulation) if the existing PATH is longer than 1024 characters. 
Need to move to special nullsoft builds which support 8192 string lengths while creating installers.

**Comment by prateekbaheti** [comment]
Fixed from version 0.1.4 onwards for windows installers.

**Comment by koushik-shetty** [comment]
This issue seems to be back in gauge 0.3.0. This install removed all my other path variables. Please Fix the issue.
```
$ gauge -v
Gauge version: 0.3.0

Plugins
-------
html-report (2.0.0)
java (0.3.4)
ruby (0.1.0)
```

**Comment by apoorvam** [comment]
@koushik-shetty We have fixed this issue in the new release v0.3.1. Please download the latest Gauge from [website](http://getgauge.io/get-started/index.html) and try it.
